symbols.xml

```diff
@@ -1220,8 +1224,8 @@
         <global-address name='process_dig' value='0x141026a03'/>
         <global-address name='cursor' value='0x140e60058'/>
         <global-address name='selection_rect' value='0x140e60068'/>
-        <global-address name='gamemode' value='0x140e60088'/>
-        <global-address name='gametype' value='0x140e6008c'/> -4
+        <global-address name='gamemode' value='0x140e6008c'/>
+        <global-address name='gametype' value='0x140e60088'/>
```

Comment:
If these were found with find-offsets, as the comment above claims, I'm not sure how they could've been switched, since gamemode is 3 in the main menu and gametype is 11. Could you double-check those? It could be that @pronvit mixed them up, since one was corrected manually IIRC.

(Also, these are duplicated above, so you can probably get rid of the lines in this section.)